tests/functional/unit_testing/test_unit_testing.py

```diff
@@ -76,11 +76,15 @@ def test_basic(self, project):
         )
         assert len(results) == 1
 
-        # Exclude unit tests with environment variable
+        # Exclude unit tests with environment variable for build command
         os.environ["DBT_EXCLUDE_RESOURCE_TYPES"] = "unit_test"
         results = run_dbt(["build", "--select", "my_model"], expect_pass=True)
         assert len(results) == 1
 
+        # Exclude unit tests with environment variable for test command
+        results = run_dbt(["test", "--select", "my_model"], expect_pass=True)
+        assert len(results) == 0
+
```

Comment:
Nice @TowardOliver !

Could you do something similar just a little earlier in the file for when `dbt test --exclude-resource-types unit_test`? You can copy-paste-modify the section that starts with "# Run build command but specify no unit tests".